src/components/editor/Editor.jsx

```diff
@@ -91,19 +79,22 @@ Editor.propTypes = {
   children: PropTypes.array,
   triggerHandleOffsetMenu: PropTypes.func,
   resetStore: PropTypes.func,
-  jwtAuth: PropTypes.object,
   location: PropTypes.object,
   resourceTemplateId: PropTypes.string,
-  history: PropTypes.object
+  history: PropTypes.object,
+  authenticationState: PropTypes.object
```

Comment:
per the comment in `LoginPanel`, we should actually take `currentSession` as a prop, instead of the `authenticationState` object.  noted on #528.